docker-hub/onboarding-faqs.md

```diff
@@ -64,6 +68,14 @@ select the **Owners** team from the drop-down menu. [Learn more](orgs.md#the-own
 
 No. Organization owners can invite users through email and also choose a team for them to join within the invite.
 
+### Can I force my organization's members to authenticate before using Docker Desktop and are there any benefits?
+
+Yes. You can [enforce sign in](../docker-hub/configure-sign-in.md) and there are benefits.
+
+Some benefits of enforcing sign in are:
```

Comment:
Maybe add this into the first sentence?

Yes. You can enforce sign in, and some of the benefits are: